sportsref/ncaaf/players.py

```diff
@@ -33,52 +33,46 @@ def getDoc(self):
     @sportsref.decorators.memoized
     def name(self):
         doc = self.getDoc()
-        name = doc('div#info_box h1:first').text()
+        name = doc('div#meta h1:first').text()
         return name
 
     @sportsref.decorators.memoized
     def position(self):
         doc = self.getDoc()
-        rawText = (doc('div#info_box p')
+        rawText = (doc('div#meta p')
                    .filter(lambda i,e: 'Position' in e.text_content())
                    .text())
-        rawPos = re.search(r'Position: (\S+)', rawText, re.I).group(1)
-        allPositions = rawPos.split('-')
-        # TODO: right now, returning just the primary position for those with
-        # multiple positions
+        rawPos = re.search(r'Position : (\S+)', rawText, re.I).group(1)
+        allPositions = rawPos.split('/')
+        # TODO: returning just the first position for those with
+        # multiple positions. Should return the last position played
```

Comment:
interesting, why do you think I should return the last position played? Regardless, as you can see, this can become a bit of an issue. In most cases positions are clear-cut, but sometimes you run into issues with guys like Jabrill Peppers. I am currently leaning towards just returning a list regardless (e.g., return ['QB'] for Brady and ['DB', 'S', 'LB'] for Sanders), but curious to hear your thoughts.